chassis/core/src/main/java/com/griddynamics/jagger/user/test/configurations/JTest.java

```diff
@@ -0,0 +1,109 @@
+package com.griddynamics.jagger.user.test.configurations;
+
+import com.griddynamics.jagger.user.test.configurations.load.JLoad;
+import com.griddynamics.jagger.user.test.configurations.termination.JTermination;
+
+/**
+ * Test configuration holder.
+ * Here a user can set termination strategy, load configuration and other parameters of a test.
+ */
+public class JTest {
+
+
+    private String id;
+    private JLoad load;
+    private JTestDescription testDescription;
+    private JTermination termination;
+
+
+    private JTest(Builder builder) {
+        this.id = builder.id;
+        this.testDescription = builder.jTestDescription;
+        this.load = builder.load;
+        this.termination = builder.termination;
+    }
+
+    public static Builder builder() {
+        return new Builder();
+    }
+
+    public static class Builder {
+        private String id;
+        private JTestDescription jTestDescription;
+        private JLoad load;
+        private JTermination termination;
+
+
+        private Builder() {
+
+        }
+
+        /**
+         * Set {@code id} for a test.
+         *
+         * @param id for a test.
+         */
+        public Builder withId(String id) {
+            this.id = id;
+            return this;
+        }
+
+        /**
+         * Set description for a test.
```

Comment:
JTestDescription is actually kind of test template, not a human-readable string. Would be nice to reflect that in the JavaDoc.